src/openshift/cluster.ts

```diff
@@ -143,6 +146,23 @@ export class Cluster extends OpenShiftItem {
             .catch((error) => Promise.reject(`Failed to login to cluster '${clusterURL}' with '${error}'!`));
     }
 
+    static async readFromClipboard() {
+        return await clipboardy.readSync();
+    }
+
+    static async ocLogin(skipConfirmation: boolean = false): Promise<string> {
+        const response = await Cluster.requestLoginConfirmation(skipConfirmation);
```

Comment:
Why are you calling it again when it's already called in login method ?

Reply:
I am calling requestLoginConfirmation() method here for command palette to check the login status.